service/internal/proctelemetry/process_telemetry_test.go

```diff
@@ -137,10 +137,10 @@ func TestOtelProcessTelemetry(t *testing.T) {
 func TestOCProcessTelemetry(t *testing.T) {
 	ocRegistry := metric.NewRegistry()
 
-	require.NoError(t, RegisterProcessMetrics(ocRegistry, noop.NewMeterProvider(), false, 0))
+	require.NoError(t, RegisterProcessMetrics(ocRegistry, noop.NewMeterProvider(), false, 0, ""))
 
 	// Check that the metrics are actually filled.
-	<-time.After(200 * time.Millisecond)
+	time.Sleep(200 * time.Millisecond)
```

Comment:
Why the change here?

Reply:
because it seemed like a better way to express that we're waiting 200 milliseconds? I can peel that off into a chore if you like.

Reply:
also uniformize with https://github.com/open-telemetry/opentelemetry-collector/pull/7998/files#diff-bbd186f2d03f0a4df651e362301bbbe615e892be0b4f3ad8931937b281b374ddR27